pages/trips/[id]/index.js

```diff
@@ -136,7 +138,38 @@ export default function DetailsPage() {
 
     mutate();
   }
+  function DisplayCountdown() {
+    const countdown = formatDistanceToNowStrict(new Date(trip.startDate), {
+      unit: "day",
+      roundingMethod: "floor",
+      addSuffix: true,
+    });
+
+    const parsedCountdown = countdown.replace(/[^0-9]/g, "");
+
+    const countdownAdjusted = formatDistanceToNowStrict(
+      new Date(trip.startDate),
+      {
+        roundingMethod: "floor",
+        addSuffix: true,
+      }
+    );
 
+    if (!countdown.includes("ago") && parsedCountdown < 30) {
+      return `. Starts ${countdownAdjusted}.`;
+    }
+    if (countdown.includes("ago") && parsedCountdown < 1) {
+      return ". Starts today!";
+    }
+  }
+  const duration = formatDistanceStrict(
+    new Date(trip.startDate),
+    new Date(trip.endDate),
+    {
+      unit: "day",
+      roundingMethod: "floor",
+    }
+  );
```

Comment:
Please see my previous comments regarding `DisplayCountdown()` function the and the points we discussed in the Daily. 💚